### ISSUE TITLE
[BUG]: Cannot find "/help/en-US/element_tabbable_role_valid.html" in accessibility-checker-engine@3.1.30

### ISSUE THREAD

**Issue by liupeig** [IBMa/equal-access#976]
### Project

a11y checker engine

### Browser

Firefox

### Operating system

MacOS

### Description

Cannot find "/help/en-US/element_tabbable_role_valid.html" in accessibility-checker-engine@3.1.30.

### Steps to reproduce

When I check a11y issue on page I get this issue "The tabbable element's role 'search' is not a widget role [Learn more]"
After clicked Learn more, the UI displayed the error like the below:
<img width="1198" alt="b1" src="https://user-images.githubusercontent.com/107132538/173365326-af397168-41c3-448d-ba77-92a0b9b1c523.png">
<img width="1183" alt="b2" src="https://user-images.githubusercontent.com/107132538/173365351-4702238a-4b09-43b5-a25c-77a59598e202.png">

**Comment by liupeig** [comment]
@shunguoy  Will we push a update package with fix for browser extension or manually update?

**Comment by eirckr** [comment]
Please update to the latest version 3.1.32, we pushed a hotfix for this issue. Thanks

**Comment by liupeig** [comment]
Verify on 3.1.32. Fixed.
